test/cpp/util/create_test_channel.cc

```diff
@@ -18,12 +18,19 @@
 
 #include "test/cpp/util/create_test_channel.h"
 
+#include <gflags/gflags.h>
+
 #include <grpc/support/log.h>
 #include <grpcpp/create_channel.h>
 #include <grpcpp/security/credentials.h>
 
 #include "test/cpp/util/test_credentials_provider.h"
 
+DEFINE_bool(grpc_test_use_grpclb_with_pick_first_lb_policy, false,
```

Comment:
That makes sense. It will be set to pick_first for all uses right now, but I could see wanting to set to round_robin in some scenario, so I changed to take a child policy string